<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheGeneral</class>
 <widget class="QDialog" name="TheGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>THE GESTION </string>
   </property>
  </widget>
  <widget class="Line" name="topLine">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="TitleAddUser">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Ajout d'un utilisateur</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>251</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="DownLayout">
    <item>
     <widget class="QPushButton" name="ValidPush">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AnnulerPush">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="InfoWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>241</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Mon Compte</string>
    </property>
   </widget>
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="SurnameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Prenom</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatutLabel_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Statut</string>
    </property>
   </widget>
   <widget class="QLabel" name="MailLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="AdressLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adresse</string>
    </property>
   </widget>
   <widget class="QLabel" name="TelLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModifInfoButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier Information</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModifMdpButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier Mot De Passe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GestionAssoButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestion De l'Asso</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nomLabel">
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nomLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="prenomLabel">
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="prenomLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="statutLabel">
      <property name="text">
       <string>statut</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="statutLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="loginLabel">
      <property name="text">
       <string>login</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="loginLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="motDePasseLabel">
      <property name="text">
       <string>mot de passe </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="motDePasseLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="telephoneLabel">
      <property name="text">
       <string>telephone</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="telephoneLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="adresseLabel">
      <property name="text">
       <string>adresse</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="adresseLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="mailLabel">
      <property name="text">
       <string>mail</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="mailLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
